torch/testing/_internal/common_methods_invocations.py

```diff
@@ -20262,6 +20262,9 @@ def __init__(
             DecorateInfo(unittest.expectedFailure, 'TestCommon', 'test_python_ref_executor'),
         ),
     ),
+    PythonRefInfo(
```

Comment:
Cool! Follow-up question on testing:

- should we make prelu (and this reference info) elementwise unary? It's a little tricky because prelu takes a weight parameter, but other elementwise unary operations take additional kwargs, too, and we probably have the test architecture to handle it
- whether we make prelu elementwise unary or not, are the additional prelu-specific inputs sufficient to validate the weight parameter is working properly? Should we add more? Maybe even adding a "reference inputs" function for prelu with extended samples?
- does prelu have ErrorInputs for its custom checks, like channel-related checks? If not, should we add them?

Reply:
Ah, I will think about these points more!

Reply:
Updated to make prelu elementwise unary, weight is always scalar in sample kwargs, but the 1-D case is tested in other sample inputs. Also added ErrorInputs.